extensions/amp-story/0.1/test/test-amp-story.js

```diff
@@ -110,7 +110,7 @@ describes.realWin(
       element.remove();
     });
 
-    it('should build with the expected number of pages', () => {
+    it.skip('should build with the expected number of pages', () => {
```

Comment:
skipping these tests since `amp-story:0.1` is deprecated and this code will be deleted soon anyways.

Reply:
We're deleting `amp-story-0.1.js`?

Reply:
Yes, it has been through the deprecation process and approved for deletion. /cc @newmuis

Reply:
Should these tests be skipped when it's being deleted? Or does this PR break the tests?

Reply:
The changes here break these tests :/

Reply:
I'll leave it to your team whether or not skipping this is ok. I wouldn't though.

Reply:
Fixed the tests and re-enabled them